sbi/src/api/tee_host.rs

```diff
@@ -73,24 +73,10 @@ pub fn reclaim_pages(addr: u64, num_pages: u64) -> Result<()> {
 ///
```

Comment:
stray (squash artifact?) comments in the commit message.

Reply:
Yes.